frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/SendNotificationHandler.java

```diff
@@ -295,6 +296,13 @@ private APIGatewayProxyResponseEvent handleNotificationRequest(
         sessionService.storeOrUpdateSession(
                 session.incrementCodeRequestCount(
                         request.getNotificationType(), request.getJourneyType()));
+
+        incrementUserSubmittedCredentialIfNotificationSetupJourney(
+                METRICS,
+                request.getJourneyType(),
+                request.getNotificationType().name(),
+                configurationService.getEnvironment());
```

Comment:
Sorry @BeckaL. Looking at this I've seen that I've not specified enough on this ticket, as `request.getJourneyType() == JourneyType.REGISTRATION` must also be `true` for this metric to be raised.

We'd also need to condition this on `notificationType == NotificationType.VERIFY_EMAIL || notificationType == NotificationType.VERIFY_PHONE_NUMBER` please, as there are other notification types that are handled by this lambda but that shouldn't have this metric raised for them.

It may be cleaner to execute `incrementUserSubmittedCredentialIfNotificationSetupJourney` within the conditions below that handle the different notification types (phone will need a condition on journey type added too).

Reply:
After discussion, we don't actually need to condition this on journey type and we already have a check of the notification type in the metrics helper